packages/pluggableWidgets/datagrid-web/src/components/__tests__/__snapshots__/Header.spec.tsx.snap

```diff
@@ -1,13 +1,13 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Header renders the structure correctly 1`] = `
```

Comment:
Should we also add a few extra unit tests to cover sizing logic?

Reply:
Definitely, will add together with the leftover removal

Reply:
Done